pkg/controllers/controllers.go

```diff
@@ -74,7 +74,7 @@ func NewControllers(
 
 	controllers := []controller.Controller{
 		p, evictionQueue, disruptionQueue,
-		disruption.NewController(clock, kubeClient, p, cloudProvider, recorder, cluster, disruptionQueue),
+		disruption.NewController(disruption.MakeConsolidation(clock, cluster, kubeClient, p, cloudProvider, recorder, disruptionQueue)),
```

Comment:
I'm curious why you want to encapsulate everything inside of consolidation and pass it to the constructor like this. If you are literally wrapping everything in the constructor, it sort of begs the question why you even need to wrap it in the first place